core/src/main/java/org/apache/accumulo/core/client/rfile/RFileWriterBuilder.java

```diff
@@ -98,7 +98,8 @@ public RFileWriter build() throws IOException {
     userProps.putAll(samplerProps);
 
     if (!userProps.isEmpty()) {
-      acuconf = new ConfigurationCopy(Iterables.concat(acuconf, userProps.entrySet()));
+      acuconf =
+          new ConfigurationCopy(Stream.concat(acuconf.stream(), userProps.entrySet().stream()));
```

Comment:
If you mean in the constructor, it's just because Stream has an iterator method already, but isn't marked with the "Iterable" interface. Using invoke direct on the stream's iterator method by declaring it as a lambda in the constructor effectively forces it to conform to the Iterable interface, but doesn't actually construct a new Iterable or do any conversion.

This is really only weird because we have so many "Iterable" usages that are leftover from before Java had Streams. As we use Streams more, and Iterables less, we'll run into fewer of these weird situations.